ci/features/profile_d.feature

```diff
@@ -3,7 +3,9 @@ Feature: profile d script
 
   @BUILD_DIR
   Scenario: Populates environment with secrets from Conjur
-    Given the 'compile' script is run
+    Given the build directory has a secrets.yml file
+    And VCAP_SERVICES contains cyberark-conjur credentials
+    And the supply script is run against the app's root folder
```

Comment:
Same comment here, should we test when the conditions are not right?

Reply:
Hm this feature appears to be identical to the buildpackless one. They use different step definitions on `master`, but they essentially boil down to the same thing. I think we should probably just remove the `buildpackless` one.